### PR TITLE
refactor/unified builder

### PR DESCRIPTION
**Pull request by scealiontach**
* fix(docker): test images now use wait-for-it
* build(make): split buildx builders among architectures
* build(make): correct manifest recipe declaration
* build: remove opa.mk references
* refactor(docker): rework unified-builder
* refactor(make): refactor so that build does not require test
* build(dockerignore): reverse declaration for simplicity

### CODE REVIEW

File: docker/unified-builder [hunk 1]
```diff
@@ -12,168 +12,233 @@
...
+  libssl-dev \
+  libzmq3-dev \
+  && apt-get install -y --no-install-recommends \
+  liblzma-dev:${TARGETARCH} \
+  libpq-dev:${TARGETARCH} \
+  libssl-dev:${TARGETARCH} \
+  libzmq3-dev:${TARGETARCH} ; \
+  fi
+
+# Finally we build the target arhitecture binaries
```
Comment (suchapalaver) [inline docker/unified-builder:106]:
```suggestion
# Finally we build the target architecture binaries
```

File: Makefile [hunk 2]
```diff
@@ -157,24 +165,22 @@ endif
 OPA_VERSION=v0.49.2
```
Comment (suchapalaver) [inline Makefile:165]:
```suggestion
OPA_VERSION=v0.50.1
```

**Review by suchapalaver — APPROVED** [review]
Can confirm this built, running  `make clean build` on my X86-64.

File: Makefile (hunk 2 above)
Comment (scealiontach) [reply Makefile:165]: That belongs in a different PR, no?

**Review by suchapalaver — APPROVED** [review]

File: .dockerignore [hunk 3]
```diff
@@ -1,241 +1,8 @@
...
-.serverless
-
-# These are backup files generated by rustfmt
-**/*.rs.bk
-
-# Project local
-
-**/sqlite_test
-**/*.env
-**/*.sqlite
```
Comment (ryanbtp) [inline .dockerignore:241]: Much neater. This was generated stuff

**Review by ryanbtp — APPROVED** [review]

**Pull request merged**